examples/webpack/karma.conf.js

```diff
@@ -0,0 +1,88 @@
+const webpack = require('webpack');
+
+const webpackConfig = {
+  module: {
+    loaders: [{
+      test: /\.ts$/,
+      loader: 'ts',
+      exclude: /node_modules/
+    }],
+    postLoaders: [{
+      test: /src\/.+\.ts$/,
+      exclude: /(test|node_modules|\.spec)/,
```

Comment:
Similar reg-ex bit us recently on a project that was called bundle-test. Took me a bit to find out why coverage was not recorded.